march_ethercat/IMC.cpp

```diff
@@ -0,0 +1,137 @@
+//
+// Class source for IMotionCubes
+// Derived from Slave class
+//
+
+#include "IMC.h"
+#include "ethercat_SDO.h"
+
+// Constructor
+IMC::IMC(std::string name, uint16 number) : Slave(name, number)
```

Comment:
Number? What does that mean? Count? Index?
I can see it's a number by its type.